VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -70,35 +70,35 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
 
     switch (branch) {
       case BranchesOfServiceConstants.AirForce:
-        return <VAIcon testID="United States Air Force" name="AirForce" {...dimensions} />
+        return <VAIcon testID="VeteranStatusUSAFIconTestID" name="AirForce" {...dimensions} />
       case BranchesOfServiceConstants.Army:
-        return <VAIcon testID="United States Army" name="Army" {...dimensions} />
+        return <VAIcon testID="VeteranStatusUSArmyIconTestID" name="Army" {...dimensions} />
       case BranchesOfServiceConstants.CoastGuard:
-        return <VAIcon testID="United States Coast Guard" name="CoastGuard" {...dimensions} />
+        return <VAIcon testID="VeteranStatusUSCoastGuardTestID" name="CoastGuard" {...dimensions} />
       case BranchesOfServiceConstants.MarineCorps:
-        return <VAIcon testID="United States Marine Corps" name="MarineCorps" {...dimensions} />
+        return <VAIcon testID="VeteranStatusUSMarineTestID" name="MarineCorps" {...dimensions} />
       case BranchesOfServiceConstants.Navy:
-        return <VAIcon testID="United States Navy" name="Navy" {...dimensions} />
+        return <VAIcon testID="VeteranStatusUSNavyTestID" name="Navy" {...dimensions} />
```

Comment:
Small nit, but I noticed the name of the testID for these icons don't include the word `Icon` like the first 2 do, e.g. `VeteranStatusUSArmyIconTestID`